packages/api/src/database.ts

```diff
@@ -169,15 +169,17 @@ export class DatabaseInstance {
       const roundSummaryData = {
         contributionCount: summary.contributionCount,
         uniqueContributors: summary.uniqueContributors,
-        totalContributionsInUSD: Number(summary.totalContributionsInUSD),
-        averageUSDContribution: Number(summary.averageUSDContribution),
+        totalContributionsInUSD: Number(summary.totalContributionsInUSD) ? Number(summary.totalContributionsInUSD) : 0,
+        averageUSDContribution: Number(summary.averageUSDContribution) ? Number(summary.averageUSDContribution) : 0,
```

Comment:
^ Why is this needed ? 
The default value for summary.totalContributionsInUSD and summary.averageUSDContribution is 0 
Why would we have anything else here ?

Reply:
I added this and forgot to remove it as a sanity check.